README.md

```diff
@@ -51,16 +51,32 @@ From the repo:
 
 ### Browser
 
-From the repository: Compile `bitcoinjs-min.js` with the following command:
+If you're familiar with how to use browserify, ignore this and proceed normally.
+These steps are advisory only and allow you to use the API to its full extent.
 
-    $ npm run-script compile
+[Browserify](https://github.com/substack/node-browserify) is assumed to be installed for these steps.
 
-From NPM:
+From your repository, create a `foobar.js` file
 
-    $ npm -g install bitcoinjs-lib browserify uglify-js
-    $ browserify -r bitcoinjs-lib -s bitcoin | uglifyjs > bitcoinjs.min.js
+``` javascript
+var foobar = {
+  base58: require('bs58'),
+  bitcoin: require('bitcoinjs-lib'),
```

Comment:
it will complain that `bitcoinjs-lib` is not found as it has not yet been installed through npm